locales/en-US/editor.properties

```diff
@@ -145,7 +145,9 @@ DND_SUCCESS_UNZIP=Successfully unzipped <b>{0}</b>.
 DND_SUCCESS_UNTAR=Successfully untarred <b>{0}</b>.
 
 # Image Viewer
-IMAGE_DIMENSIONS={0} (width) &times; {1} (height) pixels
+IMAGE_FILE_TITLE=Image File
+IMAGE_DIMENSIONS_1={0} &times; {1} pixels
```

Comment:
I think the l10n guidelines support your suggestion - create a meaningful name vs. just tacking on numbers.